gensim/corpora/wikicorpus.py

```diff
@@ -274,7 +274,16 @@ def __init__(self, fname, processes=None, lemmatize=utils.has_pattern(), diction
         if processes is None:
             processes = max(1, multiprocessing.cpu_count() - 1)
         self.processes = processes
-        self.lemmatize = lemmatize
+        # if the user has set lemmatize flag as True, check if installed
+        if lemmatize == True:
+            try:
```

Comment:
is there a reason for this code to be duplicated here and in `lemmatize`?
Just leaving it in `lemmatize` is enough